feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt

```diff
@@ -89,7 +89,7 @@ abstract class AboutFragmentModule {
         @PageScope
         @JvmStatic
         @Provides
-        fun providesLifecycleOwnerLiveData(
+        fun providesLifeCycleLiveData(
```

Comment:
Probably your commit contains other changes. 🙏

Reply:
sorry...and thanks.. I fixed